templates/security/compliance-automation.html

```diff
@@ -12,302 +16,534 @@
-        <a href="/security/certifications/docs" class="p-button--positive">Read the docs</a>
+    </div>    
+    <div class="row--50-50">
+      <hr class="p-rule--muted" />
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="row">
+            <div class="col-3">
+              <div class="p-image-container is-highlighted">
+                {{ image(url="https://assets.ubuntu.com/v1/babd31f2-fips-140-2.png",
+                                  alt="",
+                                  width="852",
+                                  height="569",
+                                  hi_def=True,
+                                  loading="lazy") | safe
+                }}
+              </div>
+              <p>
+                <a href="/security/fips">FIPS 140-2 Level 1&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+            <div class="col-3">
+              <p>These modules are NIST-certified:</p>
+              <hr class="p-rule--muted u-no-margin--bottom" />
+              <ul class="p-list--divided">
+                <li class="p-list__item is-ticked">Ubuntu 20.04 LTS</li>
+                <li class="p-list__item is-ticked">Ubuntu 18.04 LTS</li>
+                <li class="p-list__item is-ticked">Ubuntu 16.04 LTS</li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="row">
+            <div class="col-3">
+              <div class="p-image-container is-highlighted">
+                {{ image(url="https://assets.ubuntu.com/v1/9ecfeade-fips-140-3.png",
+                                  alt="",
+                                  width="852",
+                                  height="569",
+                                  hi_def=True,
+                                  loading="lazy") | safe
+                }}
+              </div>
+              <p>
+                <a href="/blog/ubuntu-22-04-fips-140-3-modules-available-for-preview">FIPS 140-3 Level 1&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+            <div class="col-3">
+              <p>
+                These modules have been assessed by a NIST-approved testing laboratory and are awaiting final certification by <a href="https://csrc.nist.gov/Projects/cryptographic-module-validation-program/modules-in-process/modules-in-process-list">CMVP</a>:
+              </p>
+              <hr class="p-rule--muted u-no-margin--bottom" />
+              <ul class="p-list--divided">
+                <li class="p-list__item is-ticked">Ubuntu 22.04 LTS</li>
+              </ul>
+            </div>
+            <hr class="p-rule--muted u-hide--medium u-hide--large" />
+            <p class="u-hide--medium u-hide--large">
+              <a href="/blog/ubuntu-22-04-fips-140-3-modules-available-for-preview">FIPS 140-3 Level 1&nbsp;&rsaquo;</a>
+            </p>
```

Comment:
I think we can remove this one since the on above already matches the link placement for 140-2
![image](https://github.com/user-attachments/assets/a903cb98-0a93-4fb1-89b9-53528ae9c2d2)

```suggestion
```